src/cpp/src/image_generation/models/autoencoder_kl.cpp

```diff
@@ -238,15 +238,18 @@ AutoencoderKL& AutoencoderKL::compile(const std::string& device, const ov::AnyMa
     OPENVINO_ASSERT(m_decoder_model, "Model has been already compiled. Cannot re-compile already compiled model");
     ov::Core core = utils::singleton_core();
 
+    std::optional<AdapterConfig> adapters;
+    auto filtered_properties = extract_adapters_from_properties(properties, &adapters);
```

Comment:
The original request was about enabling `ov_pipeline.compile(device, config=ov_config)`. Not `pipe.compile(args.text_encoder_device, args.unet_device, args.vae_decoder_device, config=properties)`. Does it work now?